### PR TITLE
Fixes #11389: Add normation based debian box in rtf

### PR DESCRIPTION
**Pull request by peckpeck**
https://www.rudder-project.org/redmine/issues/11389

### CODE REVIEW

**Comment by peckpeck** [comment]
Commit modified

**Comment by peckpeck** [comment]
Commit modified

**Comment by peckpeck** [comment]
Commit modified

File: rtf [hunk 1]
```diff
@@ -176,6 +178,22 @@ class Host:
   def __init__(self, host_info):
     self.info = host_info
 
+use_proxy = None
+def have_proxy():
```
Comment (amousset) [inline rtf]: get_proxy?

**Comment by peckpeck** [comment]
Commit modified

File: rtf (hunk 1 above)
Comment (VinceMacBuche) [reply rtf]: get_proxy is defined in vagrant.rb

**Pull request merged**